code/modules/cargo/packs.dm

```diff
@@ -1674,7 +1674,7 @@
 					/obj/item/seeds/liberty,
 					/obj/item/seeds/amanita,
 					/obj/item/seeds/reishi,
-					/obj/item/seeds/banana,
```

Comment:
🤔

Reply:
To be fair bananas are in the seed vendor already, and theres normally atleast 1 or 2 bananas lying around the station or in the corpse of the clown. There's no reason to have banana seeds in the seed crate aswell.